sources/packages/forms/src/form-definitions/exportfinancialreports.json

```diff
@@ -503,18 +550,17 @@
               "autoAdjust": false,
               "customClass": "",
               "hidden": false,
-              "hideLabel": false,
               "modalEdit": false,
               "key": "columns",
               "tags": [],
               "properties": {},
               "conditional": {
-                "show": null,
-                "when": null,
+                "show": "",
+                "when": "",
                 "eq": "",
                 "json": ""
               },
-              "customConditional": "",
+              "customConditional": "show = data['reportName'] === 'Disbursement_Forecast_Report' || data['reportName'] === 'Disbursement_Report' || data['reportName'] === 'Data_Inventory_Report'",
```

Comment:
This can be formatted as below for better readability.
```js
show = 
    data.reportName === 'Disbursement_Forecast_Report'
    || data.reportName === 'Disbursement_Report'
    || data.reportName === 'Data_Inventory_Report';
```

Reply:
I am not able to see this change. Can you please check if it was updated?

Reply:
To be clear, is there a need to access the property as `data['reportName']` instead of `data.reportName`?

Reply:
Yes @andrewsignori-aot , I see that the file is updated. I had pushed the change.

Reply:
I will change it to use a dot.